### ISSUE TITLE
Merge "sdk-network-providers" and "sdk-wallet" into "sdk-core"

### ISSUE THREAD

**Issue by andreibancioiu** [multiversx/mx-sdk-js-core#483]
The following libraries will be merged into `sdk-core`:
 - https://github.com/multiversx/mx-sdk-js-network-providers
 - https://github.com/multiversx/mx-sdk-js-wallet

Notes:
 - Git history will be kept (best effort).
 - Once merging is complete, we'll release a new minor version of **sdk-core v13** (no breaking changes).
 - We're following the example of [sdk-py](https://github.com/multiversx/mx-sdk-py) (already unified).

**Comment by danielailie** [comment]
This was done a while back, I'll close this issue as we don't need it anymore
